### ISSUE TITLE
Observe that no name is given for dark/light theme: A11y_.NET 9 template_.NET 9 template_Project_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#27760]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter.
4. Tab till theme button.
5. Observe that no name is given for dark/light theme.

### Actual Result:
Observe that no name is given for dark/light theme.

### Expected Result:
Name should be provided for dark/light theme button.

### User Impact:
Screen reader users will get the confuse if no name is provided for buttons available.

### Attachment:

https://github.com/user-attachments/assets/89814ef4-bf44-4435-a14b-c56d58fffe5d

![Image](https://github.com/user-attachments/assets/ab4b6d51-612d-4105-af8a-d02b07fd69f0)

**Comment by AnKushSingh05** [comment]
Issue is fixed. Hence closing the bug 

![Image](https://github.com/user-attachments/assets/7570cdd4-f6ed-4841-9ce1-5d3781c57aa4)

![Image](https://github.com/user-attachments/assets/ce8a0a10-3c18-4152-aa5d-d1c5f1f39b6b)
